kolibri/core/errorreports/schemas.py

```diff
@@ -0,0 +1,64 @@
+context_frontend_schema = {
+    "type": "object",
+    "definitions": {
+        "versionInfo": {
+            "type": "object",
+            "properties": {
+                "name": {"type": "string", "optional": True},
+                "major": {"type": "string", "optional": True},
+                "minor": {"type": "string", "optional": True},
+                "patch": {"type": "string", "optional": True},
+            },
+        }
+    },
+    "properties": {
+        "browser": {
+            "$ref": "#/definitions/versionInfo",
+        },
+        "component": {"type": "string", "optional": True},
+        "os": {
+            "$ref": "#/definitions/versionInfo",
+        },
+        "device": {
+            "type": "object",
+            "properties": {
+                "model": {"type": "string", "optional": True},
+                "type": {"type": "string", "optional": True},
+                "vendor": {"type": "string", "optional": True},
+                "is_touch_device": {"type": "boolean", "optional": True},
+                "screen": {
+                    "type": "object",
+                    "properties": {
+                        "width": {"type": "integer", "optional": True},
+                        "height": {"type": "integer", "optional": True},
+                        "available_width": {"type": "integer", "optional": True},
+                        "available_height": {"type": "integer", "optional": True},
+                    },
+                },
+            },
+        },
+    },
+}
+context_backend_schema = {
+    "type": "object",
+    "properties": {
+        "request_info": {
+            "type": "object",
+            "properties": {
+                "url": {"type": "string", "optional": True},
+                "method": {"type": "string", "optional": True},
+                "headers": {"type": "object", "optional": True},
+                "body": {"type": "string", "optional": True},
+            },
+        },
+        "server": {
+            "type": "object",
+            "properties": {
+                "host": {"type": "string", "optional": True},
+                "port": {"type": "string", "optional": True},
+            },
+        },
+        "packages": {"type": "object", "optional": True},
```

Comment:
I think it makes more sense for packages to be an array instead!

Reply:
Ideally following the requirements.txt  format eg
```json
["pytz==2022.1",  "django==3.2.24"]
```

Reply:
It would also help us easily determine those that are pinned and not

Reply:
Suppose we want pinned or not information. Then we need to get it from the requirements.txt file right? Or is there any other way?

Reply:
pinned versions would be `django==3.2.24` and unpinned `django`  for example, so we can tell that way

Reply:
From our discussions doesn't make much of the difference as all packages will all be of a particular version, so just return them as suggested above